From ec914626d7360db2cc8bab97b56ec9f65a6348da Mon Sep 17 00:00:00 2001
From: Andreas Gampe <agampe@google.com>
Date: Thu, 1 Mar 2018 17:42:59 -0800
Subject: [PATCH 08/68] Gallery2: Fix CollectionIncompatibleType

Fix incorrect indexOf lookup.

Bug: 73288226
Test: m javac-check RUN_ERROR_PRONE=true
Change-Id: Ifb158d6f4c5f0eb078f9d4fb37c7da03c2c0a4fe
---
 .../gallery3d/filtershow/filters/FilterGradRepresentation.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/gallery3d/filtershow/filters/FilterGradRepresentation.java b/src/com/android/gallery3d/filtershow/filters/FilterGradRepresentation.java
index 3459a8776..34d8c9488 100644
--- a/src/com/android/gallery3d/filtershow/filters/FilterGradRepresentation.java
+++ b/src/com/android/gallery3d/filtershow/filters/FilterGradRepresentation.java
@@ -104,7 +104,7 @@ public class FilterGradRepresentation extends FilterRepresentation
                 count++;
             }
         }
-        return "c=" + mBands.indexOf(mBands) + "[" + mBands.size() + "]" + count;
+        return "c=" + mBands.indexOf(mCurrentBand) + "[" + mBands.size() + "]" + count;
     }
 
     private void creatExample() {
-- 
2.17.1

